aspect/testing/tests/src/com/google/idea/blaze/aspect/java/javabinary/JavaBinaryTest.java

```diff
@@ -58,10 +58,8 @@ public void testJavaBinary() throws Exception {
             testRelative(intellijInfoFileName("foo")));
     assertThat(getOutputGroupFiles(testFixture, "intellij-resolve-java"))
         .containsAtLeast(
-            testRelative("libfoolib.jar"),
```

Comment:
why these jars are no longer expected, is optimization enabled for this test? same for `DependenciesTest`

Reply:
Reverted these changes, they were traces to when optimization was enabled by default.